GradientLoadingBar/Classes/Views/RoundedGradientActivityIndicatorView.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  CircleGradientActivityIndicatorView.swift
+//  GradientLoadingBar
+//
+//  Created by Aleksei Cherepanov on 15.11.2020.
+//
+
+import Foundation
+
+@IBDesignable
+open class RoundedGradientActivityIndicatorView: GradientActivityIndicatorView {
+    var borderWidth: CGFloat = 3 {
+        didSet {
+            setNeedsLayout()
+            layoutIfNeeded()
+        }
+    }
+
+    private var cornerRadius: CGFloat = 0
+    private var prevoiusRect: CGRect = .zero
+
+    override func commonInit() {
+        super.commonInit()
+
+        if #available(iOS 12.0, *) {
+            gradientLayer?.type = .conic
+            // It's imposible to use centered gradient in this view model configuration
+            // gradientLayer?.startPoint = CGPoint(x: 0.5, y: 0.5)
+            // gradientLayer?.endPoint = CGPoint(x: 0.5, y: 0)
+        } else {
+            // Default graidient is fine but not perfect
+        }
+    }
+
+    override open func layoutSubviews() {
+        super.layoutSubviews()
+
+        var newRadius = superview?.layer.cornerRadius ?? 0
+        newRadius = max(0, newRadius - borderWidth)
```

Comment:
Could you please explain why we have to subtract the `borderWidth` from the `cornerRadius` ?